core/chainio/avs_writer.go

```diff
@@ -93,6 +93,17 @@ func (w *AvsWriter) SendAggregatedResponse(batchIdentifierHash [32]byte, batchMe
 	txOpts.NoSend = false
 	i := 0
 
+	// Set Retry config for RespondToTaskV2
+	respondToTaskV2Config := retry.DefaultRetryConfig()
+	respondToTaskV2Config.MaxNumRetries = 0
+	respondToTaskV2Config.MaxElapsedTime = 0
+
+	// Set Retry config for WaitForTxRetryable
+	waitForTxConfig := retry.DefaultRetryConfig()
+	waitForTxConfig.MaxInterval = 2 * time.Second
+	waitForTxConfig.MaxNumRetries = 0
+	waitForTxConfig.MaxElapsedTime = timeToWaitBeforeBump
```

Comment:
Doesn't the `MaxNumRetries = 0` mean these won't retry?

Reply:
There was additional logic added to the retry functions that in the case of 0 retries the function retries indefinitely. This needs to be merged with main we moved these parameters to separate config functions.